src/Forms/GridField/GridFieldConfig_RecordEditor.php

```diff
@@ -10,8 +10,10 @@ class GridFieldConfig_RecordEditor extends GridFieldConfig
     /**
      *
      * @param int $itemsPerPage - How many items per page should show up
+     * @param bool $showPagination Whether the `Previous` and `Next` buttons should display or not, leave as null to use default
+     * @param bool $showAdd Whether the `Add` button should display or not, leave as null to use default
      */
-    public function __construct($itemsPerPage = null)
+    public function __construct($itemsPerPage = null, $showPagination = null, $showAdd = null)
```

Comment:
I think it'd be better to make this configuration rather than new constructor arguments

Reply:
It is in the constructor so that this can be customised per instance, rather than statically.

Let me know if you know a better way to approach this.

Reply:
Oh I'm sorry!

>Unlike with other methods, PHP will not generate an E_STRICT level error message when __construct() is overridden with different parameters than the parent __construct() method has.

Ignore me

Reply:
Haha, no problem. I started thinking you were correct too!